<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Information_patient</class>
 <widget class="QDialog" name="Information_patient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>491</width>
     <height>377</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="Label_13">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Імя</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LineEdit_13"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Label_14">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Прізвище</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="LineEdit_14"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Label_15">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Дата народження</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="LineEdit_15"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Label_16">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Адреса</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="LineEdit_16"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Label_17">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Номер телефону</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LineEdit_17"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Label_18">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Дата прибуття</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="LineEdit_18"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="Label">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Логін</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="LineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="Label_2">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="LineEdit_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Нотатки</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>2</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
	border-image: url(:/del/1486504830-delete-dustbin-empty-recycle-recycling-remove-trash_81361.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>35</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
	border-image: url(:/del/1486504830-delete-dustbin-empty-recycle-recycling-remove-trash_81361.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>68</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
	border-image: url(:/del/1486504830-delete-dustbin-empty-recycle-recycling-remove-trash_81361.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
	border-image: url(:/del/1486504830-delete-dustbin-empty-recycle-recycling-remove-trash_81361.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>134</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
	border-image: url(:/del/1486504830-delete-dustbin-empty-recycle-recycling-remove-trash_81361.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>168</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
	border-image: url(:/del/1486504830-delete-dustbin-empty-recycle-recycling-remove-trash_81361.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>382</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton:hover{
border-width: 0px;
	
	color: rgb(11, 11, 11);
border-radius: 20%;

}
 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
	border-image: url(:/del/1486504830-delete-dustbin-empty-recycle-recycling-remove-trash_81361.png);
	font: 75 oblique 11pt &quot;Norasi&quot;;
outline: none;
 }


 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>390</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Все</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>389</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Information_patient</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Information_patient</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
